scripts/js/queries.js

```diff
@@ -612,14 +626,17 @@ $(function () {
 
       // Prefix colored DNSSEC icon to domain text
       var dnssecIcon = "";
-      dnssecIcon =
-        '<i class="mr-2 fa fa-fw ' +
-        dnssec.icon +
-        " " +
-        dnssec.color +
-        '" title="DNSSEC: ' +
-        dnssec.text +
-        '"></i>';
+      if (doDnssec !== false) {
```

Comment:
```suggestion
      if (doDnssec === true) {
```

Double negation is harder to parse

Reply:
Yeah it is